src/libarena/lib.rs

```diff
@@ -297,24 +293,25 @@ impl<'longer_than_self> Arena<'longer_than_self> {
 
         unsafe {
             let buf = head.as_ptr();
-            (buf.offset(tydesc_start as isize), buf.offset(start as isize))
+            (buf.offset(tydesc_start as isize),
+             buf.offset(start as isize))
```

Comment:
why does this get split into two lines when there are much longer lines in the file?